sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -782,11 +782,12 @@ class StreamingAggregationSuite extends StateStoreMetricsTest with Assertions {
       testStream(aggregated, Update())(
         StartStream(checkpointLocation = tempDir.getAbsolutePath),
         AddData(inputData, 21),
-        ExpectFailure[SparkException] { e =>
+        ExpectFailure[
+          org.apache.spark.sql.execution.streaming.state.StateStoreValueSchemaNotCompatible] { e =>
```

Comment:
[nit] we can import `org.apache.spark.sql.execution.streaming.state.StateStoreValueSchemaNotCompatible` above to make this more readable.

Reply:
Done